static/app/views/performance/transactionSummary/transactionTags/content.tsx

```diff
@@ -97,7 +97,8 @@ function getTagKeyOptions(tableData: TableData) {
 const InnerContent = (
   props: Props & {tableData: TableData | null; isLoading?: boolean}
 ) => {
-  const {eventView, location, organization, tableData} = props;
+  const {eventView: _eventView, location, organization, tableData} = props;
+  const eventView = _eventView.clone();
```

Comment:
Does cloning the eventView fix the search issues you've identified?

Reply:
Yeah, it was re-arranging the tag keys on the left every time since selecting the tag key would be applied to the event view that searches for the sidebar data.